lib/utils/fsUtils.js

```diff
@@ -117,7 +129,7 @@ async function createTemplateFiles(templateType, handle, textMain, textParts) {
     );
     fs.writeFile(partPath, textParts[textPartName], emptyCallback);
   });
-  console.log(`Liquid template file(s) created for ${handle}`);
+  consola.debug(`Liquid template file(s) created for ${handle}`);
```

Comment:
Could also still be useful as info imo

Reply:
In that case it will always be shown. By default we show info. I think once you know it will create those files, you would improve your experience reducing the terminal output (it won't add any value to you)